graphql/query_resolver.go

```diff
@@ -438,15 +438,27 @@ func (r *queryResolver) Project(ctx context.Context, projectIdentifier string) (
 
 // Projects is the resolver for the projects field.
 func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error) {
+	usr := mustHaveUser(ctx)
+	viewableProjects, err := usr.GetViewableProjectSettings(ctx)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error getting viewable projects for '%s': '%s'", usr.DispName, err.Error()))
+	}
 	allProjects, err := model.FindAllMergedTrackedProjectRefs()
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
 	// We have to iterate over the merged project refs to verify if they are enabled
 	enabledProjects := []model.ProjectRef{}
 	for _, p := range allProjects {
```

Comment:
fwiw it's actually not possible for enabled to be set at the repo level anymore so i think that comment is outdated^